ci-operator/step-registry/cucushift/installer/rehearse/vsphere/ipi/remote-worker/cucushift-installer-rehearse-vsphere-ipi-remote-worker-workflow.yaml

```diff
@@ -0,0 +1,9 @@
+workflow:
+  as: cucushift-installer-rehearse-vsphere-ipi-remote-worker
+  steps: 
+    pre:
+    - chain: cucushift-installer-rehearse-vsphere-ipi-remote-worker-provision
+    post:
+    - chain: cucushift-installer-rehearse-vsphere-ipi-deprovision
```

Comment:
Can we confirm the provisioned remote workers also can be destroyed together?

Reply:
Will check it on VMC. And it should be destroyed because worker provisioned with machineset have marked with infra label.